src/m5stack_core2.toit

```diff
@@ -90,21 +91,21 @@ class Power:
     set_bits device CHARGE_CONTROL_REGISTER_1 value --mask=mask
 
   /// Sets GPIO 0 to either floating or grounded.
-  gpio0 --floating=true --grounded=(not floating) -> none:
-    assert: not floating and grounded
+  gpio0 --floating/bool=false --grounded/bool=false -> none:
+    assert: grounded == (not floating)
```

Comment:
That should be an error and not just an assert.
You should probably also state it in the toitdoc.